SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -1019,5 +1020,68 @@ private void GetScale()
+                .Description("Provides a readout of a given ship's stats.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
+                .RequiresTarget()
+                .Action((user, target, location, args) =>
+                {
+                    if (GetIsDM(target) != true && GetIsDMPossessed(target) != true)
+                    {
+                        if (Space.GetShipStatus(target) != null)
+                        {
+                            var npcStats = Stat.GetNPCStats(target);
+                            var level = npcStats.Level;
+                            if (GetIsPC(target) == true)
+                            {
+                                var playerId = GetObjectUUID(target);
+                                var dbPlayer = DB.Get<Player>(playerId);
+                                level = dbPlayer.Skills[Service.SkillService.SkillType.Piloting].Rank;
+                            }
+                            var targetName = GetName(target);
+                            var targetStatus = Space.GetShipStatus(target);
+                            SendMessageToPC(user, $"{targetName} stats: \n" +
+                                $"Armor: {targetStatus.Hull} Max: {targetStatus.MaxHull} \n" +
+                                $"Shields: {targetStatus.Shield} Max: {targetStatus.MaxShield} \n" +
+                                $"Capacitor: {targetStatus.Capacitor} Max: {targetStatus.MaxCapacitor} \n" +
+                                $"Shield Regen: {targetStatus.ShieldRechargeRate} \n" +
+                                $"EM Damage Bonus: {targetStatus.EMDamage} \n" +
+                                $"Thermal Damage Bonus: {targetStatus.ThermalDamage} \n" +
+                                $"Explosive Damage Bonus: {targetStatus.ExplosiveDamage} \n" +
+                                $"Accuracy: {targetStatus.Accuracy} \n" +
+                                $"Evasion: {targetStatus.Evasion} \n" +
+                                $"Thermal Defense: {targetStatus.ThermalDefense} \n" +
+                                $"EM Defense: {targetStatus.EMDefense} \n" +
+                                $"Explosive Defense: {targetStatus.ExplosiveDefense} \n" +
+                                $"Skill Level: {level}");
+                        }
+                    }
+                });
+        }
+
+        private void RepairShip()
+        {
+            _builder.Create("repairship")
+                .Description("Restores a ship's armor, shield and capacitor by the indicated amount.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
+                .RequiresTarget()
+                .Action((user, target, location, args) =>
+                {
+                    if (GetIsDM(target) != true && GetIsDMPossessed(target) != true)
```

Comment:
Small stylistic thing, but would prefer this to be either:

!GetIsDM(target) && !GetIsDMPossessed(target)

or

GetIsDM(target) == false && GetIsDMPossessed(target) == false

The first one is more preferred over the second, but != true is weird :D